### ISSUE TITLE
node 10.9.0

### ISSUE THREAD

**Issue by CurtisHumphrey** [jondot/hygen#68]
Using node 10.9.0 hygen just hangs after install 1 or 2 files (out of 5 in my template).  There isn't any log output it just stops (but does not exit).  Node 8.11.4 still works

**Comment by jondot** [comment]
Thanks for reporting! I'm able to reproduce this as part of the CI now. I'll check and get back to you.

**Comment by jondot** [comment]
Curtis - I find this fails only in tests, but works well with my templates.
Could you send over a sample of the problematic templates? (or maybe a sample case repository on Github that I can use?)

**Comment by jondot** [comment]
All tests now pass under latest Node version. please LMK if this still affects you

**Comment by jondot** [comment]
Just ran it:
```
~/experiments/test-curtis-hygen
❯ hygen component dumb
? Where does it belong (folder) test
? What should it be called? foo

Loaded templates: _templates
       added: src/components/test/Foo/Foo.js
       added: src/components/test/Foo/package.json
       added: src/components/test/Foo/Foo.scss
       added: src/components/test/Foo/Foo.spec.js
       added: src/components/test/Foo/Foo.stories.js
```
Sidenote - I notice you're on Windows - and your paths are forward-slash. Maybe try backslash? `\`

**Comment by CurtisHumphrey** [comment]
Yes, I'm on windows.  I just tried switching the slashes and that made no difference.  Still just hangs.  It still seams odd that it will create the first file but not the rest.  Adding breakpoints it seems the issue is somehow related to the yields not returns starting with line 35 in lib\engine.js. If I change it to `if(!fs.existsSync(actionfolder)` it will proceed to line 48 and then get stuck on that yield.

**Comment by jondot** [comment]
Just wanted to ping back and see if you've got this resolved?

**Comment by freakezoid** [comment]
Same issue here for me. Any solutions to this issue?

**Comment by jondot** [comment]
@freakezoid are you too on the same kind of machine / environment? (e.g. Windows)

**Comment by elevatebart** [comment]
Hello @jondot I have the same issue on Windows 10 with 1.6.2. I wonder if I can send you the complete config.
It seems to be connected to the behavior of yield on windows. 
Started happening recently after upgrading to Node 10. 
I wonder if it has anything to do with fs-extra.

**Comment by elevatebart** [comment]
I actually just found out that the function `exists` is not implemented by fs-extra. Instead they use `pathExists` [doc](https://github.com/jprichardson/node-fs-extra/blob/master/docs/pathExists.md)
